packages/calcite-components/src/components/list-item/list-item.tsx

```diff
@@ -703,7 +703,11 @@ export class ListItem
     const cells = [actionsStartEl, contentEl, actionsEndEl].filter(Boolean);
     const currentIndex = cells.findIndex((cell) => composedPath.includes(cell));
 
-    if (key === "Enter") {
+    if (
+      key === "Enter" &&
+      !composedPath.includes(actionsStartEl) &&
```

Comment:
Nice!

Sidebar: it would be useful to do an audit to see if other components might need this too. cc @geospatialem

Reply:
Added https://github.com/Esri/calcite-design-system/issues/7686 for a spike in next month's milestone.